packages/wonder-blocks-icon-button/components/icon-button-core.js

```diff
@@ -104,11 +110,19 @@ export default class IconButtonCore extends React.Component<Props> {
         if (href) {
             const actualHref = !disabled ? href : undefined;
             return clientNav ? (
-                <StyledLink {...commonProps} to={actualHref}>
+                <StyledLink
+                    {...commonProps}
+                    onClick={this.handleClick}
+                    to={actualHref}
+                >
                     {child}
                 </StyledLink>
             ) : (
-                <StyledAnchor {...commonProps} href={actualHref}>
+                <StyledAnchor
+                    {...commonProps}
+                    onClick={this.handleClick}
+                    href={actualHref}
```

Comment:
🛑 At least for `StyledAnchor` we always want to pass in `href` (not `actualHref`). I'm not sure if that's the case for `StyledLink` as well, maybe it is? Now that there is the `handleClick` with the `preventDefault` it seems like we can just remove `actualHref` and use `href`, correct?

Reply:
😞 whoops, I'll fix this.